### PR TITLE
Add vit isic

### PR DESCRIPTION
**Pull request by smartdanny**
- adds vit to isic
- makes vit default
- adds pl_func for clearml logger
- adds clearml logger to isic

### CODE REVIEW

File: examples/fuse_examples/imaging/classification/isic/isic_runner.py [hunk 1]
```diff
@@ -208,6 +277,8 @@ def run_train(paths: dict, train_common_params: dict) -> None:
     lgr.info("Datamodule:", {"attrs": "bold"})
 
     datamodule = create_datamodule(paths, train_common_params)
+    # datamodule.setup(stage="fit")
```
Comment (mosheraboh) [inline examples/fuse_examples/imaging/classification/isic/isic_runner.py:280]: remove

**Review by mosheraboh — COMMENTED** [review]
Thanks. Looks good!
Few comments inline.

File: fuse/dl/lightning/pl_funcs.py [hunk 2]
```diff
@@ -36,6 +36,48 @@
 from fuse.eval import MetricBase
 from fuse.eval.metrics.utils import PerSampleData
 
+# for clearml
+from clearml import Task
```
Comment (mosheraboh) [inline fuse/dl/lightning/pl_funcs.py:40]: Add clearml to dependency list

File: fuse/dl/lightning/pl_funcs.py [hunk 3]
```diff
@@ -36,6 +36,48 @@
...
+
+TaskInstance = TypeVar("TaskInstance", bound="Task")
+
+
+def start_clearml_logger(
+    project_name: Union[str, None] = None,
```
[the remark this answers is not in the document]
Comment (smartdanny) [reply fuse/dl/lightning/pl_funcs.py:46]: Are you referring to all of the arguments? Or just project name?

File: examples/fuse_examples/imaging/classification/isic/isic_runner.py [hunk 4]
```diff
@@ -194,6 +261,8 @@ def run_train(paths: dict, train_common_params: dict) -> None:
     # ==============================================================================
     # Logger
     # ==============================================================================
+    data_string = "multimodal" if multimodality else "imaging"
+    start_clearml_logger(project_name="SHATZ_isic_II", task_name=f"{model_type}_{data_string}")
```
[the remark this answers is not in the document]
Comment (smartdanny) [reply examples/fuse_examples/imaging/classification/isic/isic_runner.py:265]: yes you make a `~/clearml.conf` file and specify server address. ill dm you more details

File: fuse/dl/lightning/pl_funcs.py (hunk 2 above)
Comment (smartdanny) [reply fuse/dl/lightning/pl_funcs.py:40]: added to dependency in [31a8f07](https://github.com/BiomedSciAI/fuse-med-ml/pull/221/commits/31a8f07db0f58ae07ae20a38916fa994eafc684d)

File: fuse/dl/lightning/pl_funcs.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (smartdanny) [reply fuse/dl/lightning/pl_funcs.py]: made basic args mandatory here: [31a8f07](https://github.com/BiomedSciAI/fuse-med-ml/pull/221/commits/31a8f07db0f58ae07ae20a38916fa994eafc684d)

File: examples/fuse_examples/imaging/classification/isic/isic_runner.py (hunk 1 above)
Comment (smartdanny) [reply examples/fuse_examples/imaging/classification/isic/isic_runner.py]: removed here [8e9a212](https://github.com/BiomedSciAI/fuse-med-ml/pull/221/commits/8e9a212220e6f419a52c8ebdbdbaeef7f5ccc6d5)

**Review by mosheraboh — APPROVED** [review]

**Pull request merged**